codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/IdempotencyTokenMiddlewareGenerator.java

```diff
@@ -15,207 +15,190 @@
 
 package software.amazon.smithy.go.codegen.integration;
 
-import java.util.ArrayList;
```

Comment:
no need to comment this out, you can just remove it from the registry

Reply:
Right, updated